include/rocksdb/cache.h

```diff
@@ -47,7 +48,19 @@ enum CacheMetadataChargePolicy {
 const CacheMetadataChargePolicy kDefaultCacheMetadataChargePolicy =
     kFullChargeCacheMetadata;
 
-struct LRUCacheOptions {
+struct CacheOptions {
```

Comment:
I'm not sure, this CacheOptions is general enough. Maybe some other cache will not use shard? May need some discussion here.

Reply:
I agree that at least some of these options are implementation-specific. After #9126 I'm likely to work on a new clock cache that does not use sharding, and might not support strict_capacity_limit.

Reply:
Removed the CacheOptions